source/object-model/enumerables.md

```diff
@@ -46,11 +46,11 @@ in an observable fashion, you should use `myArray.get('firstObject')` and
 
 In the rest of this guide, we'll explore some of the most common enumerable
 conveniences. For the full list, please see the [Ember.Enumerable API
-reference documentation.](http://emberjs.com/api/classes/Ember.Enumerable.html)
+reference documentation.](https://emberjs.com/api/ember/2.15/classes/Ember.Enumerable)
```

Comment:
Hmm, this one doesn't exist at `release`, is that why some of these still reference 2.15?  @toddjordan any thoughts on whether we want to support those in the future?